samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/Application.java

```diff
@@ -94,8 +94,6 @@ public CloudAdapter getCloudAdapter(Configuration configuration) {
 
         CloudAdapterWithErrorHandler adapter = new CloudAdapterWithErrorHandler(
             configuration,
-            telemetryInitializerMiddleware,
-            botTelemetryClient,
```

Comment:
If those parameters are removed from the initialization, `telemetryInitializerMiddleware` and `botTelemetryClient` are not used anywhere. However, I think that not using those instances anywhere may make the sample to compile and run but not do what they are expected to do. Can we check this?

When the sample was updated, `AdapterWithErrorHandler` was replaced by `CloudAdapterWithErrorHandler`, but `AdapterWithErrorHandler` neither has a constructor with those parameters, however builds fine 🤷‍♀️

https://github.com/southworks/BotBuilder-Samples/blob/e1b6d70b2be013bb5656f3652350c3c3bfc8b6bb/samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/Application.java#L94-L98

Instead of just removing the parameters and hiding the problem below the carpet I'd rather find where those parameters are missing and add them. 

@enzocanoo  @FedericoBernal  can you give me a second opinion on this? I don't have extensive context about this change